packages/contracts/src/current/tokens/ZRXToken/ZRXToken.sol

```diff
@@ -16,9 +16,9 @@
 
 */
 
-pragma solidity ^0.4.11;
+pragma solidity ^0.4.19;
```

Comment:
ZRXToken should keep the same version and imports, since we're not actually going to be updating it.